isubit/luggage_roles
# LUGG-1003 Added frontpanel ds view and configuration for page

This ds view will be used in other content types, such as projects, news, and events.

To test:
- pull down this branch and revert the luggage_roles feature
- make a page
- add it to the front panel with the "Frontpanel" ds view
- Does page have categories? Does page have tags? Is there a read_more link? The answer to all these questions should be "no"
